### ISSUE TITLE
MrBayes needs ssh or rsh installed

### ISSUE THREAD

**Issue by lgorenstein** [bioconda/bioconda-recipes#27786]
Hi Bioconda team, while playing with MrBayes container, [I noticed](https://github.com/BioContainers/containers/issues/404) that it is missing `ssh` and/or `rsh` in the environment. Because of this, the MPI version of the program can not be launched:
```
$ singularity run https://depot.galaxyproject.org/singularity/mrbayes:3.2.7--h19cf415_2 mpirun -np 2 mb-mpi
--------------------------------------------------------------------------
The value of the MCA parameter "plm_rsh_agent" was set to a path
that could not be found:

  plm_rsh_agent: ssh : rsh

Please either unset the parameter, or check that the path is correct
--------------------------------------------------------------------------
[host.example.com:141273] [[INVALID],INVALID] FORCE-TERMINATE AT Not found:-13 - error plm_rsh_component.c(335)
```

The issue can be _somewhat_ mitigated by exporting `OMPI_MCA_plm_rsh_agent=""` in the parent shell, but this leads to another undesired effect: in the absence of `ssh` or `rsh`, OpenMPI can no longer launch across multiple nodes (i.e. parallel runs are only confined to cores within a single node where OpenMPI could get by using `self` mechanism).

In other words, without `ssh` or `rsh` pretty much all MPI tools are crippled.  In a regular Bioconda environment things get by because they can utilize the host's `ssh`.  But no such luck in containerized environment... 

Since this container is build from the Bioconda package, would it be possible to add `ssh` client into the upstream recipe?

**Comment by peterjc** [comment]
Do you mean add ``ssh`` as a run time dependency of the ``mrbayes`` package?

https://github.com/bioconda/bioconda-recipes/blob/master/recipes/mrbayes/meta.yaml

**Comment by lgorenstein** [comment]
Hi Peter, yes, exactly!  And just the SSH client, not the SSHD server.

To be honest, to me this looks like an exception that is only needed for containerized version (i.e. would be better handled on the Biocontainers' side rather than a Bioconda one).  But Biocontainers developers think that doing it in the conda recipe may be better/easier (please see [the discussion in bioontainers](https://github.com/BioContainers/containers/issues/404) I linked earlier).
